run_tests_coverage.sh

```diff
@@ -11,8 +11,16 @@ mode_desc=( [1]="use shelves" [2]="no shelves" )
 while [ $count -le $numOfModes ]
 do
     echo "Run with ${mode_desc[$count]}"
-    coverage run -a -m pytest pdsfile/pds3file/tests/ \
-    pdsfile/pds3file/rules/*.py pdsfile/pds4file/tests/ pdsfile/pds4file/rules/*.py --mode ${mode_arr[$count]}
+    # Only run with use shelves for pds3, we don't have shelves for pds4 for now
```

Comment:
This file has gotten really complicated. How about just having four separate test lines with the appropriate arguments and skipping all the arrays and loops and conditionals?